docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
+
+## General Environment Setup
+
+### Declare Variables
+
+Some global variables used throughout the overlay setup.
+
+- `ZITI_HOME` is the directory where your network files will be stored, the directory will be created if it does not exist already.
+- `ZITI_BIN_DIR` is the directory where the ziti binary will be downloaded and extracted
+- `ZITI_PWD` is the password that will be used when logging into the controller as the default user (admin).
+- `ZITI_HOSTNAME` is used throughout the setup for naming files, network elements, etc.
+
+```
+export ZITI_HOME="${HOME}/.ziti"
+export ZITI_BIN_DIR="${ZITI_HOME}/ziti_bin"
+export ZITI_PWD=admin2
+export ZITI_HOSTNAME=$(hostname -s)
+```
+
+### Create Environment
+
+Create a directory where you will store your network files.
```

Comment:
I'd change the 'you/your' here. Feels out of place. Try to be critical about using 'you/your' and remove it when possible.

Also more words, maybe with the outline this will feel more natural. Would be good to explain how the ZITI_HOME is set if the user doesn't set it.